src/components/ConnectorIcon.tsx

```diff
@@ -0,0 +1,34 @@
+import QuestionMarkIcon from '@mui/icons-material/QuestionMark';
+import { Box } from '@mui/material';
+
+interface Props {
+    size?: number;
+    iconPath: string | undefined;
+}
+
+const defaultSize = 20;
+
+function ConnectorIcon({ size = defaultSize, iconPath }: Props) {
```

Comment:
Breaking the icon to a stand alone component so we can use it on the connector autocomplete input.